services/src/floodlight-modules/src/checkstyle/checkstyle-suppress-known-issues.xml

```diff
@@ -173,4 +173,12 @@
     <suppress files="src/test/java/org/openkilda/floodlight/FloodlightTestCase.java" lines="109" checks="JavadocMethod"/>
     <suppress files="src/test/java/org/openkilda/floodlight/FloodlightTestCase.java" lines="118" checks="AbbreviationAsWordInName"/>
     <suppress files="src/test/java/org/openkilda/floodlight/FloodlightTestCase.java" lines="118" checks="JavadocMethod"/>
+    <suppress files="src/test/java/org/openkilda/floodlight/test/standard/OutputCommands.java" lines="85" checks="JavadocMethod"/>
```

Comment:
It’s agreed that we don’t add suppressions anymore instead we fix the issues.

Reply:
Add javaDocs for my method, and remove supress